config/locales/en.yml

```diff
@@ -214,12 +214,14 @@ en:
       one: 'Subject'
       other: 'Subjects'
     leadership_level: 'Leadership level'
-    job_count_without_search: 'There is %{count} job listed'
-    job_count_plural_without_search: 'There are %{count} jobs listed'
-    job_count: '%{count} job matches your search'
-    job_count_plural: '%{count} jobs match your search'
-    job_count_with_location_category: '%{count} job match near %{location}'
-    job_count_plural_with_location_category: '%{count} jobs match near %{location}'
+    job_count_without_search: There is <span class='govuk-!-font-weight-bold'>%{count}</span> job listed
+    job_count_plural_without_search: "There are <span class='govuk-!-font-weight-bold'>%{count}</span> jobs listed"
```

Comment:
because these keys contain HTML, they should end in `_html` like:

`job_count_plural_without_search_html` so that you do not need to use the method `raw` in the views

Reply:
👍

Reply:
the raw thing caught me out for an hour or two :D

Reply:
it seems that you still need the raw method in the views, but i have changed all these to _html anyway as it makes it clear